### ISSUE TITLE
The loss doesn't decrease

### ISSUE THREAD

**Issue by chun-chun123** [STVIR/pysot#260]
When I run the 'train.py' with only one GPU and load pretrained model provided,the loss maintains 0.8-0.9.
How can I make it decrease?I try to decrease batchsize but it seems useless.
![image](https://user-images.githubusercontent.com/56161480/70759153-5d815d80-1d80-11ea-9772-8ddfa6d8f498.png)

**Comment by yc-zc** [comment]
Maybe consider adjusting your learning rate.

**Comment by xyl-507** [comment]
@ZhiyuanChen
Hi! If I want to inrease my epoch to 40, can you give some advice to adjust the learning rate ? 
Thanks, in advance!
